app/controllers/green_lanes/check_your_answers_controller.rb

```diff
@@ -15,10 +15,42 @@ def new
       @answers = check_your_answers_params[:ans]
       @c1ex = check_your_answers_params[:c1ex]
       @c2ex = check_your_answers_params[:c2ex]
+      @back_link_path = determine_back_link_path(check_your_answers_params)
     end
 
     private
 
+    def determine_back_link_path(permitted_params)
+      cat_2_questions_exist = !permitted_params[:ans]['2'].nil?
+
+      if all_exemptions_met?(1, @category_one_assessments, @answers) && cat_2_questions_exist
```

Comment:
why does the backlink need to know if all exemptions are met?

Reply:
Yeah thinking about this now maybe they could be removed im trying to remember why i put them there. i guess if cat 2 questions exist then fine can go to them